### PR TITLE
Multiple GitHub org support

### PR DESCRIPTION
**Pull request by amulobaev**

### CODE REVIEW

File: components/server/src/github/github-auth-provider.ts [hunk 1]
```diff
@@ -82,26 +82,40 @@ export class GitHubAuthProvider extends GenericAuthProvider {
...
+
+            const orgs = org.split(',').map(x => x.trim());
+
+            for (const orgName of orgs) {
+                log.info(`fetchCurrentOrg org: ${orgName}`);
+                try {
+                    const response = await api.orgs.getMembershipForAuthenticatedUser({ org: orgName });
+                    log.info(`Status: ${response.status}, State: ${response.data.state}`);
+                    if (response.status === 200 && response.data.state === 'active') {
+                        return response.data.state;
```
Comment (pavan-tri) [inline components/server/src/github/github-auth-provider.ts:101]: this should be {state:"active"}; ?  otherwise we are just returning string

**Review by pavan-tri — CHANGES_REQUESTED** [review]

File: components/server/src/github/github-auth-provider.ts (hunk 1 above)
Comment (amulobaev) [reply components/server/src/github/github-auth-provider.ts:101]: Hi @pavan-tri 
Original code:
image

File: components/server/src/github/github-auth-provider.ts [hunk 2]
```diff
@@ -82,26 +82,33 @@ export class GitHubAuthProvider extends GenericAuthProvider {
...
-            const response = await api.orgs.getMembershipForAuthenticatedUser({
-                org,
-            });
-            if (response.status !== 200) {
-                throw new GitHubApiError(response);
-            }
-            const {state} = response.data;
-            if(state != "active"){
-                throw new GitHubApiError(response);
+            if (!org) return {state:"active"};
```
Comment (pavan-tri) [inline components/server/src/github/github-auth-provider.ts:85]: Test with blank org.  If the return is string, this is a bug i introduced . 
if (!org) return {state:"active"}; => if (!org) return "active";

**Review by pavan-tri — APPROVED** [review]

File: components/server/src/github/github-auth-provider.ts (hunk 2 above)
Comment (amulobaev) [reply components/server/src/github/github-auth-provider.ts]: Tested, no issues

**Pull request merged**